verification/dependencies/github.com/google/gopacket/writer.gobra

```diff
@@ -102,17 +87,17 @@ type SerializeBuffer interface {
 	decreases
 	Clear() (err error)
```

Comment:
The spec for this method should also have `preserves sl.Bytes(UBuf(), 0, len(UBuf()))`